resqpy/rq_import/_import_vdb_all_grids.py

```diff
@@ -0,0 +1,87 @@
+"""_import_vdb_all_grids.py: Module to import a vdb into resqml format."""
+
+version = '15th November 2021'
+
+# Nexus is a registered trademark of the Halliburton Company
+
+import logging
+
+log = logging.getLogger(__name__)
+log.debug('_import_vdb_all_grids.py version ' + version)
+
+import resqpy.olio.vdb as vdb
+# import resqpy.olio.grid_functions as gf
+
+from resqpy.rq_import import import_nexus
+
+
+def import_vdb_all_grids(
+    resqml_file_root,  # output path and file name without .epc or .h5 extension
+    extent_ijk = None,  # 3 element numpy vector applicable to ROOT
+    vdb_file = None,
+    vdb_case = None,  # if None, first case in vdb is used (usually a vdb only holds one case)
+    corp_xy_units = 'm',
+    corp_z_units = 'm',
+    corp_z_inc_down = True,
+    ijk_handedness = 'right',
+    geometry_defined_everywhere = True,
+    treat_as_nan = None,
+    resqml_xy_units = 'm',
+    resqml_z_units = 'm',
+    resqml_z_inc_down = True,
+    shift_to_local = False,
+    local_origin_place = 'centre',  # 'centre' or 'minimum'
+    max_z_void = 0.1,  # vertical gaps greater than this will introduce k gaps intp resqml grid
+    split_pillars = True,
+    split_tolerance = 0.01,  # applies to each of x, y, z differences
+    vdb_static_properties = True,
+    # if True, static vdb properties are imported (only relevant if vdb_file is not None)
+    vdb_recurrent_properties = False,
+    decoarsen = True,
+    timestep_selection = 'all',
+    # 'first', 'last', 'first and last', 'all', or list of ints being reporting timestep numbers
+    create_property_set = False):
+    """Creates a RESQML dataset containing grids and grid properties, including LGRs, for a single realisation."""
```

Comment:
We will need to add the full argument list to the docstring here as well – most can be copied from one of the other functions.